couchbase/metadata.csv

```diff
@@ -313,4 +313,36 @@ couchbase.sync_gateway.shared_bucket_import.import_high_seq,gauge,,,,The highest
+couchbase.index.cache_hits,gauge,,,,Accesses to this index data from RAM,0,couchbase,
+couchbase.index.cache_misses,gauge,,,,Accesses to this index data from disk,0,couchbase,
+couchbase.index.data_size,gauge,,byte,,The size of indexable data that is maintained for the index or partition,0,couchbase,
+couchbase.index.disk_size,gauge,,byte,,Total disk file size consumed by the index or partition,0,couchbase,
+couchbase.index.frag_percent,gauge,,,,Percentage fragmentation of the index,0,couchbase,
+couchbase.index.initial_build_progress,gauge,,,,"Percentage initial build progress for the index. When initial build is completed, the value is 100. For an index partition, the value is listed as 0",0,couchbase,
+couchbase.index.items_count,gauge,,,,The number of items currently indexed,0,couchbase,
+couchbase.index.last_known_scan_time,gauge,,nanosecond,,"Time of the last scan request received for this index (Unix timestamp in nanoseconds). This may be useful for determining whether this index is currently unused. Note: This statistic is persisted to disk every 15 minutes, so it is preserved when the indexer restarts",0,couchbase,
+couchbase.index.memory_used,gauge,,unit,,description,0,couchbase,
+couchbase.index.num_docs_indexed,gauge,,,,Number of documents indexed by the indexer since last startup,0,couchbase,
+couchbase.index.num_docs_pending,gauge,,,,Number of documents pending to be indexed,0,couchbase,
+couchbase.index.num_docs_queued,gauge,,,,Number of documents queued to be indexed,0,couchbase,
+couchbase.index.num_items_flushed,gauge,,,,Number of items flushed from memory to disk storage,0,couchbase,
+couchbase.index.num_pending_requests,gauge,,,,Number of requests received but not yet served by the indexer,0,couchbase,
+couchbase.index.num_requests,gauge,,,,Number of requests served by the indexer since last startup,0,couchbase,
+couchbase.index.num_rows_returned,gauge,,,,Total number of rows returned so far by the indexer,0,couchbase,
+couchbase.index.num_scan_errors,gauge,,,,Number of requests that failed due to errors other than timeout,0,couchbase,
+couchbase.index.num_scan_timeouts,gauge,,,,"Number of requests that timed out, either waiting for snapshots or during scan in progress",0,couchbase,
+couchbase.index.recs_in_mem,gauge,,,,"For standard index storage, this is the number of records in this index that are stored in memory. For memory-optimized index storage, this is the same as items_count",0,couchbase,
+couchbase.index.recs_on_disk,gauge,,,,"For standard index storage, this is the number of records in this index that are stored on disk. For memory-optimized index storage, this is 0",0,couchbase,
+couchbase.index.resident_percent,gauge,,,,Percentage of the data held in memory,0,couchbase,
+couchbase.index.scan_bytes_read,gauge,,,,Number of bytes read by a scan since last startup,0,couchbase,
+couchbase.index.total_scan_duration,gauge,,,,Total time spent by the indexer in scanning rows since last startup,0,couchbase,
```

Comment:
```suggestion
couchbase.index.total_scan_duration,gauge,,,,The total time spent by the indexer scanning rows since the last startup,0,couchbase,
```

Reply:
@ruthnaebeck Sorry about that. Should be all addressed now. Thanks for checking!